portfolio/src/main/webapp/index.html

```diff
@@ -1,113 +1,132 @@
-    <link href='https://fonts.googleapis.com/css?family=Aclonica:400,300,300italic,400italic,600,600italic,700,700italic,800,800italic' rel='stylesheet' type='text/css'>
-    <link href='https://fonts.googleapis.com/css?family=Adamina:400,100,200,300,500,600,700,800,900' rel='stylesheet' type='text/css'>
-
     <script src="script.js"></script>
-
   </head>
   <body>
-
-      <!-- Navigation Bar -->
-      <nav class="navbar navbar-default top">
-          <div class="container text-center">
-              <div class="navbar-header top">
-                  <div data-toggle="collapse" data-target="#myNavbar">
-                      <span class="sr-only">Toggle navigation</span>
-                      <span class="icon-bar"></span>
-                      <span class="icon-bar"></span>
-                      <span class="icon-bar"></span> 
-                  </div>
-              </div>
-
-    <div id="navbar" class="navbar-collapse">
-        <ul class="nav navbar-nav">
+    <!-- Navigation Bar -->
+    <nav class="navbar navbar-default top">
+      <div class="container text-center">
+        <div class="navbar-header top">
+          <div data-toggle="collapse" data-target="#myNavbar">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span> 
+          </div>
+        </div>
+        <div id="navbar" class="navbar-collapse">
+          <ul class="nav navbar-nav">
             <li><a href="index.html">Home</a></li>
-            <li><a href="resume.pdf">Resume</a></li>
+            <li><a href="documents/resume.pdf">Resume</a></li>
             <li><a href="mailto:shivani.ghanta@gmail.com">Email Me</a></li>
-
-        </ul>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <!-- Title Content & Bio -->
+    <div class="bitmoji-top">
+      <div class="container">
+        <div class="row">
+          <div class="about">
+            <div class="bio">
+              <h1>Shivani Ghanta</h1>
+              <h3>Aspiring Software Engineer</h3>
+              <hr class="top-bio">
+              <br>
+              <p>
+                I’m an incoming junior majoring in Computer Science
+                at the University of Illinois at Urbana-Champaign.<br>
+                I'm currently an Engineering STEP Intern at Google.<br>
+              </p>
+            </div>
+          </div>
+          <div class="about">
+            <div id="image-bitmoji">
+              <img src="images/bitmoji.PNG" height="430" width="430"/>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <!-- Courses Taken -->
+    <div class="info-row">
+      <div class="container">
+        <div class="row">
+          <div class="about">
+            <center>
+              <h2>Courses</h2>
+              <p class="struct">
+                Data Structures and Algorithms (CS 225)<br>
```

Comment:
Can you use an unordered list instead? (`<ul><li>`)

The bullet point can be hidden using CSS, and this way, you'd be achieving a similar behavior while using the intended tag(s) for listing items.

Reply:
When I use an unordered list, it changes the font and size even though it's in the same class and I'm not sure why.

Reply:
I suspect it's because you're setting a CSS style for `<p>` and/or `struct`, and it doesn't apply to the bullet points. You could define your own style / class for the bullet points, and use the same styling as `<p>`?

Reply:
I tried setting the same style as <p> and struct for the unordered list class but somehow the font is still different. I think there might be some inherent styling for unordered lists.

Reply:
Sure! I added a TODO comment for each list